03.redis-postgres/.ruby-version

```diff
@@ -1 +1 @@
-2.2.1
+ruby-2.3.0
```

Comment:
You should be fine with just `2.3.0` and as `2.3.1` is already available, don't we want to switch to that directly?